src/widgetastic/widget/base.py

```diff
@@ -873,11 +874,13 @@ def a_method(self):
     """
     #: Skip this view in the element lookup hierarchy
     INDIRECT = False
+    fill_strategy = None
 
     def __init__(self, parent, logger=None, **kwargs):
         Widget.__init__(self, parent, logger=logger)
         self.context = kwargs.pop('additional_context', {})
         self.last_fill_data = None
+        self.fill_strategy = BasicFillViewStrategy()
```

Comment:
I don't like this pattern, and am wondering if there's some specific need for it? If I subclass View and set my own fill_strategy class attribute, once that view class is instantiated, my locally defined fill_strategy will be overridden.